.github/workflows/land-blocking.yml

```diff
@@ -11,19 +11,72 @@ jobs:
     name: Build images and run cluster test
     runs-on: self-hosted
     steps:
-    - uses: actions/checkout@v1
-    - name: Setup env
-      run: |
-        echo "::set-env name=LIBRA_GIT_REV::$(git rev-parse --short=8 HEAD)"
-    - name: Build, tag and push images
-      run: |
-        docker/build-aws.sh --build-all --version $LIBRA_GIT_REV --addl_tags canary
-    - name: Launch cluster test
-      run: |
-        JUMPHOST=${{secrets.CLUSTER_TEST_JUMPHOST}} \
-        ./scripts/cti \
-          --marker land \
-          --tag dev_$LIBRA_GIT_REV \
-          --report report.json \
-          --run bench \
-          -- --duration 60
+      - uses: actions/checkout@v1
+      - name: Setup env
+        run: |
+          echo "::set-env name=LIBRA_GIT_REV::$(git rev-parse --short=8 HEAD)"
+      - name: Build, tag and push images
+        run: |
+          docker/build-aws.sh --build-all --version $LIBRA_GIT_REV --addl_tags canary
+      - name: Launch cluster test
+        run: |
+          JUMPHOST=${{secrets.CLUSTER_TEST_JUMPHOST}} \
+          ./scripts/cti \
+            --marker land \
+            --tag dev_$LIBRA_GIT_REV \
+            --report report.json \
+            --run bench \
+            -- --duration 60
```

Comment:
I thought a little bit more about it - maybe lets remove `-- --duration 60` for now? I could make separate PR for that, but since you are already touching this piece, do you think you could remove it?

Reply:
SG.  Will remove it.

Reply:
Fxied.